ABloom/ABloom/Presentation/CoreViews/QnA/View/QuestionMainView.swift

```diff
@@ -56,15 +33,74 @@ struct QuestionMainView: View {
             AnswerCheckView(num: content)
           }
         })
-        
-        .frame(maxWidth: .infinity, maxHeight: .infinity)
         .background(backWall())
       }
       .background(backgroundDefault())
     }
   }
 }
 
+extension QuestionMainView {
+  private var headerView: some View {
+    // 헤더
+    HStack {
+      Text("우리의 문답")
+        .fontWithTracking(.title3Bold)
+        .padding([.leading], 20)
+      Spacer()
+      // FIXME: 다른 뷰로 전환 시에는 다른 방식으로 처리해야함
+      NavigationLink(value: 0) {
+        Image("pencil_write_fill")
+      }
+      .padding([.trailing], 17)
+    }
+    .padding([.top], 20)
+    .foregroundStyle(.stone700)
+  }
+  
+  private var answeredQScroll: some View {
+    // 질문 목록
+    ScrollView(.vertical) {
+      LazyVStack(spacing: 30) {
+        
+        Spacer()
+          .frame(height: 0)
+        
+        ForEach(1..<3) { num in
+          NavigationLink(value: num) {
+            // TODO: 해당 정보란을 데이터로 가져와야함
+            QnAListItem(categoryImg: "squareIcon_isometric_sofa", question: "나와 결혼을 결심한 순간은 언제야?\(num)", date: "2023년 9월 18일", isAns: false)
+          }
+        }
+      }
+    }
+    
+  }
+  
+  private var emptyListView: some View {
+    VStack {
+      VStack {
+        (Text("오른쪽 상단의  ") + Text(Image("pencilInText")) + Text("  버튼을 눌러\n첫번째 문답을 작성해보세요."))
+          .fontWithTracking(.calloutR, tracking: -0.4)
+          .foregroundStyle(.stone400)
+          .multilineTextAlignment(.center)
+      }
```

Comment:
```suggestion
VStack(spacing: 6) {
    HStack(spacing: 7) {
        Text("오른쪽 상단의")
        Image("camera.filled.circle_outline")
            .resizable().frame(width: 20, height: 20)
        Text("버튼을 눌러")
    }
    Text("첫번째 문답을 작성해보세요.")
}
.fontWithTracking(.calloutR, tracking: -0.4)
.foregroundStyle(.stone400)
```
오프셋으로 수정할 수 있는 방법이 없다면, 수정 제안합니다!

Reply:
적용했습니다!

Reply:
감사함다!!